### PR TITLE
Feature/update domain model

### PR DESCRIPTION
**Pull request by K0F**
fixes bod c 1 from #10

### CODE REVIEW

**Comment by K0F** [comment]
..trochu bordylek, nakumulovalo se mi tam vicero commitu, ale rad bych to sloucil s mainem

File: src/cro/geneea/sdk/_domain.py [hunk 1]
```diff
@@ -95,20 +107,32 @@ def analysis(self) -> FullAnalysis:
...
+        """
         _tags: List[Tag] = []
 
         for tag in self.analyzed["tags"]:
             _tags.append(Tag(tag["id"], tag["stdForm"], tag["type"], tag["relevance"]))
         return tuple(_tags)
 
     def relations(self) -> tuple[Relation]:
+        """
+        Function returns a tuple of Realtions from analyzed JSON.
```
Comment (groundf) [inline src/cro/geneea/sdk/_domain.py:133]: Change `tuple of Relations` to `relations`. The type is already listed in the type signature.

File: src/cro/geneea/sdk/_domain.py [hunk 2]
```diff
@@ -123,13 +147,31 @@ def relations(self) -> tuple[Relation]:
...
+        """
+        Functions which returns Sentiment object from analyzed JSON.
+        :return: Sentiment object
+        """
         tmp = self.analyzed["docSentiment"]
         return Sentiment(tmp["mean"], tmp["label"], tmp["positive"], tmp["negative"])
 
+    def account(self) -> Account:
+        """
+        Function returns Account object from Analyzed JSON.
```
Comment (groundf) [inline src/cro/geneea/sdk/_domain.py:166]: Remove the word `function`, this is a method. Maybe write simply `Returns account object from analyzed JSON.`

**Review by groundf — CHANGES_REQUESTED** [review]
Write tests for the account method.

**Comment by groundf** [comment]
Please rebase against `main` (U r 1 commit behind).

File: src/cro/geneea/sdk/_domain.py (hunk 2 above)
Comment (K0F) [reply src/cro/geneea/sdk/_domain.py]: ok diky zaneseno

**Review by K0F — COMMENTED** [review]
ok doresim ty konfilkty a zkusim mergnout

**Pull request merged**